src/domains/pendingProjectAuthorization/repository/pendingProjectAuthorization.repository.impl.ts

```diff
@@ -1,19 +1,48 @@
 import { type PendingProjectAuthorizationRepository } from '@domains/pendingProjectAuthorization/repository/pendingProjectAuthorization.repository';
-import { PendingProjectAuthorizationDTO, type PendingProjectInputDTO } from '@domains/pendingProjectAuthorization/dto';
+import { PendingProjectAuthorizationDTO } from '@domains/pendingProjectAuthorization/dto';
 import type { PendingProjectAuthorization, PrismaClient } from '@prisma/client';
 import type { ITXClientDenyList } from '@prisma/client/runtime/library';
+import { type AuthorizationRequest } from '@domains/integration/dto';
+import { encryptData } from '@utils';
 
 export class PendingProjectAuthorizationRepositoryImpl implements PendingProjectAuthorizationRepository {
   constructor(private readonly db: PrismaClient | Omit<PrismaClient, ITXClientDenyList>) {}
 
-  async create(data: PendingProjectInputDTO): Promise<PendingProjectAuthorizationDTO> {
+  async create(input: AuthorizationRequest): Promise<PendingProjectAuthorizationDTO> {
     const pendingProject: PendingProjectAuthorization = await this.db.pendingProjectAuthorization.create({
       data: {
-        providerProjectId: data.providerProjectId,
-        token: data.token,
-        issueProviderId: data.issueProviderId,
-        integratorId: data.integratorId,
-        organizationId: data.organizationId,
+        integratorId: input.integratorId,
+        providerProjectId: input.projectId,
+        token: encryptData(input.apiToken),
+        members: {
+          createMany: {
+            data: input.members.map((member) => {
+              return { email: member.email };
+            }),
+          },
+        },
```

Comment:
Service that is using this method is used in an integration controller endpoint. In that endpoint you are using a validateRequest middleware, which has a constraint that validates organization and issueProvider already exists. As a result, connect action will never be used. It will work all the same but I leave a note here in case you haven't realized